ubuntu/16.04/README.md

```diff
@@ -118,10 +122,30 @@ alter any file in the codebase.
 
 As such, only set APP_USER_LOCAL in development when using volumes.
 
-### Custom Startup Scripts
+### Custom build and startup scripts
+
+To run commands during the build and startup sequences that the base images add,
+please add `usr/local/share/container/plan.sh` for a project, or
+`usr/local/share/container/baseimage-{number}.sh` if creating another base image.
+
+This allows you to define and override bash functions that the base images add.
+
+This base image adds the following bash functions:
+
+function | desciption | executed on
+do_build | By default does nothing in this image
+do_start_supervisord | Runs do_start and do_supervisord
+do_supervisord | Runs [supervisord](#SupervisorD) | do_start_supervisor
+do_start | Runs the following bash functions | do_start_supervisor
```

Comment:
the following ones :) do_update_permissions, do_development_start, do_templating